palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/model/AuditablePalisadeRequest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.palisade.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * This class is a container for {@code PalisadeResponse} and {@code AuditErrorMessage} during stream processing.
```

Comment:
Typo, should be @code PalisadeResponse

Reply:
I'm not sure what the typo is here. Are you referring to the braces `{}`?

Reply:
Realised the silly mistake, updated to `@PalisadeRequest`